docs/content/commands/npm-pkg.md

```diff
@@ -76,6 +76,14 @@ Returned values are always in **json** format.
     npm pkg get contributors[0].email
     ```
 
+    For complex fields you can also use text indices in square brackets
+    to specifically select an child field. This is especially helpful
```

Comment:
```suggestion
    to specifically select a child field. This is especially helpful
```

Reply:
That's what I get for mentally translating `an array` to `a child field` w/o proofreading.